assets/js/theme/global/modal.js

```diff
@@ -138,6 +147,7 @@ export class Modal {
         this.onModalClose = this.onModalClose.bind(this);
         this.onModalClosed = this.onModalClosed.bind(this);
 
+        this.unbindEvents();
```

Comment:
Why do we need this unbind?

Reply:
When page loaded, modal class create instances for all modals and binding events on constructor. After that this instances cashing and using until page will be reloaded. But I am thinking about if in some cases instances can be recreated and set up event listeners again and double them. I did not find scenario when it can happen. But decided to turn off old listeners before setting new, just in case

Reply:
I believe we could skip this for now, modals should not be initialised separately

Reply:
To be honest I also think that unbind should not be useful - removed